shoop/core/order_creator/_source.py

```diff
@@ -404,6 +420,13 @@ def get_validation_errors(self):
             for error in payment_method.get_validation_errors(source=self):
                 yield error
 
+        for supplier in self.get_suppliers():
+            for product, quantity in iteritems(self.get_products_and_quantities(supplier)):
+                shop_product = product.get_shop_instance(shop=self.shop)
+                for error in shop_product.get_orderability_errors(
+                        supplier=supplier, quantity=quantity, customer=self.customer):
+                    yield error
+
```

Comment:
IMHO the checks from `BaseBasket.get_validation_errors` should be moved here too, since now there is two places that check very similar things and there is no clear reason why other things are checked on the BaseBasket side and others on the OrderSource.